.github/workflows/infracost.yaml

```
@@ -0,0 +1,60 @@
+name: Infracost exibe estimativa de custo da infraestrutura
+
+on:
+  workflow_call:
+    inputs:
+      OS_VERSION:
+        description: "Versão do sistema operacional"
+        required: false
+        default: "ubuntu-latest"
```

```suggestion
        default: "ubuntu-20.04"
```

Usar `latest` pode causar quebra na pipeline quando a versão muda. `20.04` é o que a gente usa nas outras, mas talvez pode usar a mais nova `22.04` também.